libmscore/score.cpp

```diff
@@ -1767,26 +1767,30 @@ void Score::removeAudio()
 
 bool Score::appendScore(Score* score, bool addPageBreak, bool addSectionBreak)
       {
-      if (parts().size() < score->parts().size() || staves().size() < score->staves().size())
+      if (parts().size() < score->parts().size() || staves().size() < score->staves().size()) {
+            qDebug("Score to append has %d parts and %d staves, but this score only has %d parts and %d staves.", score->parts().size(), score->staves().size(), parts().size(), staves().size());
             return false;
-      TieMap tieMap;
+      }
 
-      MeasureBase* lastMeasure = last();
-      if (!lastMeasure)
+      if (!last()) {
+            qDebug("This score doesn't have any MeasureBase objects.");
```

Comment:
I added a couple qDebug statement to help me determine why my test is failing now...